pkg/build/builder/util.go

```diff
@@ -4,6 +4,11 @@ import (
 	buildapi "github.com/openshift/origin/pkg/build/api"
 )
 
+const (
+	// DefaultDockerLabelNamespace is the key of a Build label, whose values are build metadata.
+	DefaultDockerLabelNamespace = "io.openshift."
```

Comment:
@bparees @jhadvig in that case I would move this constant to API as it might be re-used in other places adding image labels.